src/management_api/proxy_services/process_model_proxy_service.ts

```diff
@@ -152,6 +154,20 @@ export class ProcessModelProxyService implements ManagementAPIs.IProcessModelMan
     name: string,
     payload: ManagementApiDataModels.ProcessModels.UpdateProcessDefinitionsRequestPayload,
   ): Promise<void> {
+
+    const parser: Parser = new Parser();
```

Comment:
Wir benutzen Type Inference.

```suggestion
    const parser = new Parser();
```